fixtures/assign.rb.sem.uast

```diff
@@ -0,0 +1,35 @@
+{ '@type': "module",
+   '@role': [Identifier, Module, Statement],
+   file: { '@type': "lvasgn",
+      '@token': "a",
```

Comment:
Should be a `uast:Identifier`

Reply:
I'm fixing this right now, will be in next commit, but I'm thinking that this (turning a type token into an identifier) is a common enough case that we should have some kind of configuration or wrapper to easily define them in the `normalizer.go` file without having to specify the roles again, something like:

```go
RoleIsIdentifier(["class", "module", "lvasgn",...]),
```.

Reply:
That's why in the current wrapper I wrote you specify the field that is the token along with the type.

Reversibility was never a requirement of SDK.v2, just high level objects, so we can tackle that later if needed.